### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by or-else**
1. Chatbot container more configurable
2. Removed `TNPG_COMPRESS_PAYLOADS` as unnecessary (always true)
3. Automatic gofmt and misc stuff.

### CODE REVIEW

File: docker/chatbot/Dockerfile [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 LABEL maintainer="Tinode Team <info@tinode.co>"
 LABEL name="TinodeChatbot"
 LABEL version=$VERSION
 
 RUN mkdir -p /usr/src/bot
+
 WORKDIR /usr/src/bot
 
+# Volume with login cookie. Not created automatically.
+# VOLUME /botdata
```
Comment (aforge) [inline docker/chatbot/Dockerfile]: What is this for?

File: docker/tinode/Dockerfile [hunk 2]
```diff
@@ -95,9 +92,6 @@ ENV TNPG_AUTH_TOKEN=
 # Tinode Push Gateway user name.
 ENV TNPG_USER=
 
-# Enable Tinode Push Gateway request payload compression.
```
Comment (aforge) [inline docker/tinode/Dockerfile]: Can you remove https://github.com/tinode/chat/blob/devel/docker/tinode/config.template#L114 as well please?

File: docker/chatbot/Dockerfile (hunk 1 above)
Comment (or-else) [reply docker/chatbot/Dockerfile]: It's a reminder which volumes have special meaning and can be mapped to external directories. I can just leave a comment without the VOLUME part but it seems clearer this way.

File: docker/tinode/Dockerfile (hunk 2 above)
Comment (or-else) [reply docker/tinode/Dockerfile]: done

**Pull request merged**